terraform/services/bucket-access-log/README.md

```diff
@@ -0,0 +1,12 @@
+# Terraform for bucket-access-logs in target accounts
```

Comment:
Please modify the directory name for the service to "bucket-access-logs" as well.